src/UbidotsESP8266.cpp

```diff
@@ -48,53 +48,53 @@ FUNCTIONS TO MANAGE DATA
  * This function is to read the Ubidots' command via serial
  */
 void Ubidots::readData() {
-
+    bool flag = false;
```

Comment:
please add a comment that explains the main purpose of this flag. Also, a more detailed variable name should be used.